go/vt/tableacl/tableacl.go

```diff
@@ -105,6 +105,7 @@ func (tacl *tableACL) init(configFile string, aclCB func()) error {
 	if configFile == "" {
 		return nil
 	}
+	log.Infof("Reading ACL file %v", configFile)
```

Comment:
I don't think we should log it here. What's the reason you need it here? Was it not clear that this was happening? I would imagine it was clear from the lines below on errors what the cause of the problem was?

Below here, there's also `Info` logging on errors which is wrong, those should be raised at least to warning if not error. It's confusing those are set at the info level. 

If there's a good reason to add the logging, maybe instead we should log inside the SIGHUP handler to make it specific for the scenario? I don't think this line is something useful to print on each startup then. We should be careful with not adding too much logging noise that loses signal then.

Reply:
> I don't think we should log it here. What's the reason you need it here? Was it not clear that this was happening? I would imagine it was clear from the lines below on errors what the cause of the problem was?

The issue is that when there is no error, sending a SIGHUP doesn't cause any log output to be generated. So, you don't know if it even did anything.

I considered adding the line to the SIGHUP handler. We can do that too I suppose. I'll change it, along with the sign-off.

Reply:
> The issue is that when there is no error, sending a SIGHUP doesn't cause any log output to be generated. So, you don't know if it even did anything.

Let's log it then after it succeeds, so that we only have the line if there was a successful reload due to a `SIGHUP`. We can then make it more something like "Successfully reloaded ACL".

Reply:
> you have no idea what it was trying to do.

I'm not sure what you mean here? This is something that logging can do properly? Looking at the code more I think it would benefit from some additional refactor and remove the exit at a distance kinda behavior and use error returns and let the callers decide.

Reply:
> I'm not sure what you mean here?

I think we're miscommunicating a bit. You said:

> What's the reason you need it here? Was it not clear that this was happening? I would imagine it was clear from the lines below on errors what the cause of the problem was?

Indeed, on error. I meant that as a sysop, I would like to see a program respond to something. If I send it a SIGHUP and no log lines are printed, I can't tell whether it did something or not. 

> Looking at the code more I think it would benefit from some additional refactor and remove the exit at a distance kinda behavior and use error returns and let the callers decide.

OK, I can do that.

Reply:
@wiebeytec https://github.com/wiebeytec/vitess/compare/main...dbussink:vitess:acl-refactor-suggestion?expand=1 is kinda what I was thinking about here. Dunno what you think of that?

Reply:
Looks good. In the branch I just force-pushed I still logged in the callees (deep), but functionality it's similar.

You can remove my code in favor of yours then.

Reply:
I think logging at the higher level would be preferable here.